packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -232,10 +249,15 @@ export class Cluster extends Resource implements ICluster {
       this._executeCommandConfiguration = props.executeCommandConfiguration;
     }
 
+    if (props.managedStorageConfiguration) {
+      this._managedStorageConfiguration = props.managedStorageConfiguration;
+    }
```

Comment:
```suggestion
    this._managedStorageConfiguration = props.managedStorageConfiguration;
```
We can avoid the `if` statement.